apis/duck/unstructured_test.go

```diff
@@ -70,23 +73,87 @@ func TestFromUnstructuredFooable(t *testing.T) {
 		want:      FooStatus{},
 		wantError: nil,
 	}}
+
 	for _, tc := range tcs {
-		raw, err := json.Marshal(tc.in)
-		if err != nil {
-			panic("failed to marshal")
-		}
+		t.Run(tc.name, func(t *testing.T) {
+			raw, err := json.Marshal(tc.in)
+			if err != nil {
+				panic("failed to marshal")
```

Comment:
I find panicing in unit-test at least quaint.

Reply:
🙄 pre-existing condition.  fixed this as well.